src/GT.Core/Services/Impl/GTListingService.cs

```diff
@@ -167,21 +167,22 @@ public async Task DeleteAsync(string id)
 		public async Task<List<ListingOverviewDTO>> GetAsync(IListingFilterModel? filter = null)
 		{
 			// Get entities from database
+
 			if (filter is null)
 			{
 				filter = new ListingFilterModel();
 			}
 
-			var query = _listingRepository
-				.GetAll()
+			var query = _listingRepository?
+				.GetAll()?
 				.Include(e => e.ExperienceLevel)
 				.Include(e => e.Location)
 				.Include(e => e.Employer)
 
 				.Where(e => filter.ExperienceLevels == null
 					|| filter.ExperienceLevels.Count <= 0
-					|| (e.ExperienceLevel != null && (e.ExperienceLevel.Name != null
-					&& filter.ExperienceLevels.Contains(e.ExperienceLevel.Name))))
+					|| (e.ExperienceLevel != null && e.ExperienceLevel.Name != null
+						&& filter.ExperienceLevels.Any(el => string.Equals(e.ExperienceLevel.Name, el))))
```

Comment:
Should this be an EF.Functions.Like() method?

Reply:
As of right now there are no user submitted strings to be compared with experience levels as there is no text input for it. That being said, it could possibly be more efficient to use EF.Functions.Compare() instead. Will do some research on the matter and possibly fix in a future pull request.